src/js/components/stock-movement-wizard/outbound/PackingPage.jsx

```diff
@@ -463,6 +470,11 @@ class PackingPage extends Component {
             <AlertMessage show={this.state.showAlert} message={this.state.alertMessage} danger />
             { !showOnly ?
               <span className="buttons-container">
+                <FilterInput
+                  itemFilter={itemFilter}
+                  onChangeFunc={e => this.setState({ itemFilter: e.target.value })}
+                  clearFunc={() => this.setState({ itemFilter: '' })}
```

Comment:
Are you sure it is working correctly in all cases? `setState` is asynchronous function, so I am thinking about situation when you input a value, then clear the value and you can see that the things that you inputted appears in the field again, because of that `clearFunc` finished its changing state earlier than `onChangeFunc`

Reply:
I get your point, but I think you are overcomplicating simple events with the same, not complexed logic applied. In "normal world" it is not possible, and this is how everyone does the clearing of form/inputs. 
Have you ever experienced anything like this and have any proposed solution? You could also pull my branch and try to achieve that :smile: I couldn't achieve that.
Answering your question - yes, I made sure it is working correctly.

Reply:
Maybe you're right that I am overcomplicating this 😄 User has to be faster than light to be able to reproduce that